README.md

```diff
@@ -269,6 +269,7 @@ minima:
     telegram: jekyll
     microdotblog: jekyll
     keybase: jekyll
+    medium: jekyll
```

Comment:
BTW there is Jekyll profile but it is unrelated to the Jekyll we know

https://medium.com/@jekyll

Reply:
Yeah, there is no official Jekyll Medium account (I think). If I go to https://micro.blog/jekyll, it is a non-existent member. If I go to https://keybase.io/team/jekyll, it is privatized. 

I figured the precedent is to use it as a placeholder.

Reply:
I think that is fine. It is to demonstrate